### DISCUSSION TITLE
Plot at 100% but not showing up in Farming

### DISCUSSION THREAD

**Discussion by Christopherkynn** [Chia-Network/chia-blockchain#2699] [Q&A]
My plot shows at 100% but not showing up in Farming. I noticed it continues to say plotting under the plot completion bar for the plot. I don't want to assume anything here. Took me awhile to to get to this point. 

Any advice would be greatly appreciated

Christopher

**Comment by Lotsofusetdesktops** [comment]
Same here today. I just let the machine alone and it started to farm the plot if few hours.

**Reply by Lotsofusetdesktops** [reply]
I had 1.0.5 version then. Upated now.

**Comment by Christopherkynn** [comment]
Thank you - I am now farming - It says it will take 5 years to mine. So..

**Reply by OversizedWizard** [reply]
People think one plot is going to do something.
I have 28 plots so far and telling me two months pmsl

Plus, how can the app say how long it'll take find a reward? It has to know when you going to farm a plot in that case.

1-1000 plots, people should get same chance.

And 28 plots, very little pass the farming filter. 

Sonu ask what's the point of plotting if the damn app isn't going to farm the plots.
